release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -81,9 +82,16 @@ With the replacement of Terraform, the following permissions are required if you
 
 For more information on required permissions, see xref:../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure].
 ====
+
+[id="ocp-4-17-installation-and-update-gcp-service-account_{context}"]
+==== Installing a cluster on {gcp-full} by using an existing service account
+
+With this update, you can install a cluster on {gcp-short} by using an existing service account, allowing you to minimize the permissions that you grant to the service account the installation program uses. You can specify this service account in the `compute.platform.gcp.serviceAccount` and `controlPlane.platform.gcp.serviceAccount` parameters in the `install-config.yaml` file. For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters_installation-config-parameters-gcp[Available installation configuration parameters for GCP].
```

Comment:
As for the "_For more information, see..._", is it possible to also tell the link of "Required GCP permissions for user-provided service accounts" (see PR https://github.com/openshift/openshift-docs/pull/81586)? Thanks!

Reply:
I should be able to add this xref once the other PR merges.